tests/unit/test_config.py

```diff
@@ -123,3 +124,23 @@ def test_user_shadows_system():
 def test_get_from_system():
     assert config.get('system_var_1') == 'system_val_1'
     assert config.get('system_var_2') == 'system_val_2'
+
+
+@mock.patch('os.environ', {})
+@mock.patch.dict('lago.config._cache', {})
+@mock.patch('lago.config._SYSTEM_CONFIG_DIR', '')
+@mock.patch('lago.config._USER_CONFIG', '')
+def test_get_only_defaults():
+    result = {k: config.get(k) for k in config.DEFAULTS.get('lago').keys()}
+    assert (
```

Comment:
Wouldn't it be sufficient to use `assert config.DEFAULTS.get('lago') == result` ?